Description: Improve error message when pdf program is missing
Author: Chris Lamb <lamby@debian.org>
Author: Jonas Smedegaard <dr@jones.dk>
License: GPL-3+
Last-Update: 2018-09-01
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/Text/Pandoc/Error.hs
+++ b/src/Text/Pandoc/Error.hs
@@ -110,7 +110,7 @@
     PandocSyntaxMapError s -> s
     PandocFailOnWarningError -> "Failing because there were warnings."
     PandocPDFProgramNotFoundError pdfprog ->
-        pdfprog <> " not found. Please select a different --pdf-engine or install " <> pdfprog
+        pdfprog <> " not found. Please select a different --pdf-engine or install " <> pdfprog <> " -- see also /usr/share/doc/pandoc/README.Debian"
     PandocPDFError logmsg -> "Error producing PDF.\n" <> logmsg
     PandocXMLError fp logmsg -> "Invalid XML" <>
         (if T.null fp then "" else " in " <> fp) <> ":\n" <> logmsg
